src/routes/Routes.tsx

```diff
@@ -1,14 +1,21 @@
 import React, { FC, memo, Suspense } from "react";
 import { Switch } from "react-router-dom";
-import Route from "./RouteWrapper";
+import {
+  setPermissions,
+  PermissionEnum,
+} from "features/permissions/permissions";
+import RouteWrapper from "./RouteWrapper";
 import { ROUTE_LIST } from "./Routes.config";
 
 const Routes: FC = () => {
+  // Set user's permission scope
+  setPermissions([PermissionEnum.DASHBOARD]);
```

Comment:
not sure placing it here at top-level is good idea - back to my original comment on storage for permissions. why not use Redux/Context state with a default value for this? then your permissions infrastructure could listen for state changes and react to permission changes without needing to manually check.

Reply:
I totally agree, but as we haven't implemented the structure for redux yet, then I didn't want to spend time using Context. I agree that we should move it to redux. I will add a ticket for it 👍